wa/instruments/energy_measurement.py

```diff
@@ -536,5 +538,9 @@ def extract_metrics(self, context):
                 context.add_metric(name, value, units)
 
     def teardown(self, context):
+        logger.debug('Job status {}'.format(context.current_job.status))
```

Comment:
This should not be logged by the instrument. This should be logged by the Runner, or possibly by the job itself, at the relevant point.